test/smoke/FirefoxPrivateVPNUITest/FirefoxPrivateVPNUITest/Tests/ServerSelectTest.cs

```diff
@@ -0,0 +1,152 @@
+﻿// <copyright file="ServerSelectTest.cs" company="Mozilla">
+// This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// </copyright>
+
+namespace FirefoxPrivateVPNUITest
+{
+    using System;
+    using System.Net;
+    using System.Threading;
+    using FirefoxPrivateVPNUITest.Screens;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+    /// <summary>
+    /// This is to test server selection.
+    /// </summary>
+    [TestClass]
+    public class ServerSelectTest
+    {
+        private FirefoxPrivateVPNSession vpnClient;
+        private BrowserSession browser;
+        private DesktopSession desktop;
+
+        /// <summary>
+        /// Initialize vpn client, browser, and desktop sessions.
+        /// </summary>
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            this.browser = new BrowserSession();
+            this.vpnClient = new FirefoxPrivateVPNSession();
+            this.desktop = new DesktopSession();
+        }
+
+        /// <summary>
+        /// Dispose vpn session, browser and desktop sessions.
+        /// </summary>
+        [TestCleanup]
+        public void TestCleanup()
+        {
+            this.vpnClient.Dispose();
+            this.browser.Dispose();
+            this.desktop.Dispose();
+        }
+
+        /// <summary>
+        /// Test server selection before user turns on VPN.
+        /// </summary>
+        [TestMethod]
+        public void TestServerSelectionBeforeConnection()
+        {
+            // Switch to VPN client session
+            this.vpnClient.Session.SwitchTo();
+            LandingScreen landingScreen = new LandingScreen(this.vpnClient.Session);
+            landingScreen.ClickGetStartedButton();
+
+            // User Sign In via web browser
+            UserCommonOperation.UserSignIn(this.vpnClient, this.browser);
+
+            // Main Screen
+            this.vpnClient.Session.SwitchTo();
+            MainScreen mainScreen = new MainScreen(this.vpnClient.Session);
+            mainScreen.ClickServerListButton();
+
+            // Server Screen
+            ServerListScreen serverListScreen = new ServerListScreen(this.vpnClient.Session);
+            serverListScreen.RandomSelectDifferentCityServer("Miami");
```

Comment:
Are we selecting Miami every time simply because it is a "stable connection"?  And I suppose currently, only servers in Miami are being tested?

Reply:
We selected Miami and Atalanta as our testing servers because Mullvad public API will return the same city name as we select.